x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/overview_logic.ts

```diff
@@ -191,7 +179,7 @@ export const OverviewLogic = kea({
       (isFederatedAuth: boolean) => (isFederatedAuth ? 'halves' : 'fourths'),
     ],
   }),
-  listeners: ({ actions }: IListenerParams) => ({
+  listeners: ({ actions }): Partial<IOverviewActions> => ({
     initializeOverview: async ({ http }: { http: HttpSetup }) => {
```

Comment:
Using Partial<> helps us check that we only define functions listed in IOverviewActions:

```suggestion
  listeners: ({ actions }): Partial<IOverviewActions> => ({
    blah: async ({ http }: { http: HttpSetup }) => {
```

`Object literal may only specify known properties, and 'blah' does not exist in type 'Partial<IOverviewActions>'.`